portal/src/texts/profile/animation.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Animasjon
+path: /profil/animasjon
+order: 8
+---
+
+# Animasjon
+
+<Ingress>
+    <p>Ved interaksjon tar vi i bruk animasjon som et virkemiddel for å understøtte våre designprinsipper og knytte brukeropplevelsen opp til disse.</p>
+    <p>Dette er knappetilstander som hover, fokus, aktiv, valgt og deaktivert eller overganger hvor elementer fjernes eller legges til. 
+    Dette forsterker brukerens handlinger underveis i løsningen.</p>
+</Ingress>
+
+## Hvorfor?
+Vi bruker animasjon for å:
```

Comment:
Må du ikke ha empty line etter overskrift i markdown for at det skal bli overskrift?

Reply:
Nei: https://www.markdownguide.org/basic-syntax/#headings